pandas/core/reshape/pivot.py

```diff
@@ -472,6 +473,7 @@ def _convert_by(by):
 
 @Substitution("\ndata : DataFrame")
 @Appender(_shared_docs["pivot"], indents=1)
+@deprecate_nonkeyword_arguments(version=None, allowed_args=["data"])
```

Comment:
Same?

Reply:
We are always using version=None, not sure why though. So wanted to be consistent here